src/current/v24.1/molt-fetch.md

```diff
@@ -91,9 +89,34 @@ Complete the following items before using MOLT Fetch:
 			~~~
 
 		- Ensure the Google Cloud Storage bucket is created and accessible to CockroachDB.
-	
+
+## Best practices
+
+- To prevent connections from prematurely terminating during data export, set the following to high values on the source database:
+
+	- **Maximum allowed number of connections.** MOLT Fetch can export data across multiple connections. The number of connections it will create is the number of shards ([`--export-concurrency`](#global-flags)) multiplied by the number of tables ([`--table-concurrency`](#global-flags)) being exported concurrently.
+	- **Maximum lifetime of a connection.** This is particularly important for MySQL sources, which can only use a single connection to move data. See the following note.
+
+- If a MySQL database is set as a [source](#source-and-target-databases), the [`--table-concurrency`](#global-flags) and [`--export-concurrency`](#global-flags) flags **cannot** be set above `1`. If these values are changed, MOLT Fetch returns an error. This is required in order to guarantee consistency when moving data from MySQL, due to MySQL limitations. MySQL data is migrated to CockroachDB one table and shard at a time, using [`WITH CONSISTENT SNAPSHOT`](https://dev.mysql.com/doc/refman/8.0/en/commit.html) transactions.
+
+## Commands
+
+| Command |                                               Usage                                               |
+|---------|---------------------------------------------------------------------------------------------------|
+| `fetch` | Start the fetch process. This loads data from a source database to a target CockroachDB database. |
+
+### Subcommands
+
+The following subcommands are run after the `fetch` command.
```

Comment:
Do you mean that the following subcommands are supported by the `fetch` command? It sort of sounds like they happen automatically when you run `fetch` the way it is now.